api/build.gradle

```diff
@@ -467,24 +468,27 @@ dependencies {
     implementation 'com.google.apis:google-api-services-cloudresourcemanager:v3-rev20220925-2.0.0'
     implementation 'com.google.apis:google-api-services-oauth2:v2-rev20200213-2.0.0'
     implementation 'com.google.apis:google-api-services-iam:v1-rev20220825-2.0.0'
-    implementation 'com.google.api-client:google-api-client-appengine:2.5.0'
-    implementation 'com.google.auth:google-auth-library-appengine:1.23.0'
-    implementation 'com.google.auth:google-auth-library-oauth2-http:1.23.0'
+    implementation 'com.google.api-client:google-api-client-appengine:2.7.1'
```

Comment:
This was mainly searching for "auth", seeing which packages were referenced in error messages, and trial and error.